Lib/defcon/objects/layoutEngine.py

```diff
@@ -38,15 +38,15 @@ def _layoutEngineOTLTablesRepresentationFactory(layoutEngine):
             addOpenTypeFeatures(feaPath, otf)
         except:
             import traceback
-            print traceback.format_exc(5)
+            print(traceback.format_exc(5))
         finally:
-            # XXX clean up file on disk
+            os.close(fd)
             os.remove(feaPath)
-        if otf.has_key("GDEF"):
```

Comment:
Ha! I wonder when I started defcon. My private SVN repo is long gone, but my guess is that it was sometime around then.